docs/get-started/HA.md

```diff
@@ -0,0 +1,666 @@
+# Set up PMM in HA mode
```

Comment:
we should make it clear that HA is in tech preview state now.

Reply:
Will add a Note

Reply:
Added a note.